libtransmission/peer-mgr.cc

```diff
@@ -2427,12 +2427,6 @@ namespace connect_helpers
 /* is this atom someone that we'd want to initiate a connection to? */
 [[nodiscard]] bool is_peer_candidate(tr_torrent const* tor, tr_peer_info const& peer_info, time_t const now)
 {
-    // have we already tried and failed to connect?
-    if (auto const conn = peer_info.is_connectable(); conn && !*conn)
```

Comment:
seperately: the logic in `get_reconnect_interval_secs` looks like it sets the actual reconnect to 15min if there was a handshake failure.  ie on the first failure it backs off to 15mins.  That seems overly conservative, no?

Reply:
I don't feel comfortable with tweaking this at the moment. Not saying that your suggestion is invalid, but there's not enough evidence showing that there is a problem.

Reply:
I'm raising because it's a problem among "long duration" private trackers, ie trackers with a lot of torrents but low seed-torrent ratio.  fwiw, the vastly more aggressive libtorrent clients (qBt, Deluge which both mindlessly retry every 60sec) somewhat unfairly grab download slots in small swarms more effectively than Transmission because of this.

Reply:
Maybe raise a new issue for it, and we continue there. This is out of the scope of this PR.